Watchman.Discord/Areas/Protection/Controllers/AdministrationController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services.Factories;
+using Watchman.Discord.Areas.Protection.BotCommands;
+using Watchman.Discord.Areas.Protection.Services;
+
+namespace Watchman.Discord.Areas.Protection.Controllers
+{
+    public class AdministrationController : IController
+    {
+        private readonly MessagesServiceFactory _messagesServiceFactory;
+        private readonly ComplaintsChannelService _complaintsChannelService;
+
+        public AdministrationController(MessagesServiceFactory messagesServiceFactory, ComplaintsChannelService complaintsChannelService)
+        {
+            this._messagesServiceFactory = messagesServiceFactory;
+            this._complaintsChannelService = complaintsChannelService;
+        }
+
+        [AdminCommand]
+        public async Task CreateChannelForComplaints(ComplaintsChannelCommand command, Contexts contexts)
+        {
+            var complaintsChannelName = string.IsNullOrWhiteSpace(command.Name) ? "skargi" : command.Name;
```

Comment:
`"skargi`
chyba lepiej ustawiać to w konfiguracji

Reply:
tylko pytanie w jakiej konfiguracji
bo "skargi" to ma być taka domyślna nazwa dla kanału jeśli ktoś nie podał innej
myślałem jeszcze czy nie usunąć wgl domyślnej wartości i wymuszać aby zawsze admin podał nazwę

Reply:
no ale w konfiguracji są przecież domyślne wartości 

robisz configuration item <string> 
jest domyślne "skargi"
ale jak ktoś chce to może zmienić

Reply:
tym bardziej że dodatnie tego do konfiguracji to nie jest za bardzo problem
bo robi sie to dość łatwo

Reply:
ale nazwę to każdy admin może sobie ręcznie zmienić i nie potrzebuje do tego bota